### ISSUE TITLE
Checking dict keys fails

### ISSUE THREAD

**Issue by kellrott** [go-python/gpython#58]
In standard python
```python
>>> x={"hello" : "world"}
>>> "hello" in x
True
```
In gpython
```python
>>> x={"hello" : "world"}
>>> "hello" in x
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
    FIXME line of source goes here
TypeError: "unsupported operand type(s) for iter: 'dict'"
```

**Comment by ncw** [comment]
Implementing `__iter__` for dict is very useful.

However for `in` to be efficient we should be implementing `__contains__` 

https://github.com/go-python/gpython/blob/61059b469f5b520f350d47748e8e9e9b6ccd3306/py/sequence.go#L121-L147

**Comment by drew-512** [comment]
suggest @sbinet close -- addressed in https://github.com/go-python/gpython/pull/65

**Comment by sbinet** [comment]
fixed w/ #65 .
